File: sources/java/basic/08/assignment/Hito.java
public class Hito {
    public static void main(String[] args) {
        Person p1 = new Person();

        int temp = p1.add(1, 2);
        System.out.println("add = " + temp);
    }
}

class Person {
    String name;
    int age;

    void show() {
        System.out.println("名前は、" + name + "です。");
        System.out.println("年齢は、" + age + "歳です。");
    }

    int getage() {
        return age;
    }

    void future_age(int year) {
        age += year;
        System.out.println(year + "年後の年齢は、" + age + "歳です。");
    }

    int add(int x, int y) {
        return (x + y);
    }

}

File: contents/python/basic/08/prog/Hito.py
class Person:
    def main(self):
        self.name = '都立大'
        self.age = 20

    def show(self):
        print('名前は、' + self.name + 'です。')
        print('年齢は、' + str(self.age) + '歳です。')

    def getage(self):
        return self.age

    def future_age(self, year):
        self.age += year
        print(year, '年後の年齢は、', self.age, '歳です。')

p1 = Person()
p1.name = 'なまえ'
p1.age = 20
p1.show()
p1.future_age(5)


